gnucash/register/register-core/pricecell.c

```diff
@@ -103,12 +117,18 @@ gnc_price_cell_modify_verify (BasicCell *_cell,
             !g_unichar_isalpha (uc) &&
             (decimal_point != uc) &&
             (thousands_sep != uc) &&
-            (g_utf8_strchr (toks, -1, uc) == NULL))
+            (g_utf8_strchr (tokens, -1, uc) == NULL))
+        {
+            g_free (tokens);
             return;
+        }
```

Comment:
The above looks a lot like the code in formulacell.c. Can you extract a function to basiccell.c and call it in both places?

Reply:
Factored out common code.